core/domain/exp_services.py

```diff
@@ -1723,6 +1723,33 @@ def revert_version_history(
         new_version_history_model.put()
 
 
+def get_exploration_version_valid_info(exploration_id, revert_to_version):
```

Comment:
```suggestion
def get_exploration_validation_error(exploration_id, revert_to_version):
```

I think this name might be more descriptive?